### DISCUSSION TITLE
I reinstalled Chrome and now not all notes are resynced

### DISCUSSION THREAD

**Discussion by yardstudio** [penge/my-notes#349] [General]
Hi, 

I have previosuly installed My Notes and synced to Google. When I reinstalled Chrome, deactivated extensions and again reactivated sync, not all notes are resync to My Notes..

**Comment by carlosmonero** [comment]
Me too and don't is sync with Maxthon, Opera etc.

**Reply by penge** [reply]
Hi @carlosmonero,
Google Drive Sync is working only in Google Chrome.

**Comment by penge** [comment]
@yardstudio 
Is there something special about the notes that haven't been synced? Like they were edited meanwhile on the PC, or in Google Drive, or both? How would you explain some notes were synced while other weren't?

**Comment by carlosmonero** [comment]
Pavel,
Its extension is very good to restrict Google Chrome. Chrome uses a lot of memory, while other alternative browsers do not use. I would like you to make the extension available in other browsers. It's possible?
Thanks,
Carlos Monero
Conectado 19/04/2022 06:00:25, Pavel Bucka ***@***.***> escreveu:
Hi @carlosmonero [https://github.com/carlosmonero],
Google Drive Sync is working only in Google Chrome.
—
Reply to this email directly, view it on GitHub [https://github.com/penge/my-notes/discussions/349#discussioncomment-2591439], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AQZRKMV5NRHJUDF2ZCBBC33VFZY2NANCNFSM5O4N2SMQ].
You are receiving this because you were mentioned.Message ID: ***@***.***>
[b6639e0f-9f71-4179-885e-803415e8210e]

**Comment by penge** [comment]
@carlosmonero 
extension can work in other browsers too, but _Google Drive Sync_ is limited to <ins>**Google Chrome only**</ins>, because of used APIs which are available in <ins>**Google Chrome only**</ins>. As **Google Drive** is Google's product, this limitation is kind of understandable and while access to **Google Drive** from other browsers work, it's different when it's accessed from an extension.

**Comment by carlosmonero** [comment]
Pavel, thank you very much for your attention and explanations.
Even so, congratulations for the product.
Conectado 19/04/2022 17:09:44, Pavel Bucka ***@***.***> escreveu:
@carlosmonero [https://github.com/carlosmonero]
extension can work in other browsers too, but Google Drive Sync is limited to Google Chrome only, because of used APIs which are available in Google Chrome only. As Google Drive is Google's product, this limitation is kind of understandable and while access to Google Drive from other browsers work, it's different when it's accessed from an extension.
—
Reply to this email directly, view it on GitHub [https://github.com/penge/my-notes/discussions/349#discussioncomment-2595981], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AQZRKMVQ5WCKQYHCKS3JNUDVF4HILANCNFSM5O4N2SMQ].
You are receiving this because you were mentioned.Message ID: ***@***.***>
[7ffb166a-ef25-4622-b594-dd37d56c5bb0]

**Comment by ulastatlidil** [comment]
I wish the notes we selected and sent with the right click were recorded at the bottom, not at the top. So it would be a top-down text.

keşke seçtiğimiz ve sağ tuş ile gönderdiğimiz notlar en üstte değil de en altta kayıt olsa. Böylece yukardan aşağıya bir metin olurdu.
